src/Umbraco.Infrastructure/Examine/ExamineExtensions.cs

```diff
@@ -75,8 +75,7 @@ public static IEnumerable<PublishedSearchResult> ToPublishedSearchResults(this I
                             content = snapshot.Media.GetById(contentId);
                             break;
                         case IndexTypes.Member:
-                            content = snapshot.Members.GetById(contentId);
-                            break;
+                            throw new NotSupportedException("Cannot convert search results to member instances");
```

Comment:
I guess we need to support this use case still? It make sense to be allowed for search for members on the website?

Reply:
It's not about searching for members, you can always search for members. This is about converting search results into IPublishedContent instances. I forgot this existed in v8 and is a horrible O(N^2) type of mess. If you do this it will be horrible for performance. Listing a ton of member data can be done just by using the search results but converting each search result one at a time to IPublishedContent requires resolving IMember one at at time too. There can be unlimited members, this really shouldn't be supported.